question-generation/main.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import sys
+import uvicorn
+from fastapi import FastAPI
+from fastapi.responses import JSONResponse
+
+from backend import get_contexts_and_questions
+
+app = FastAPI()
+
+
+@app.get("/generate")
```

Comment:
Why not have the context come from the client?

Sophia probably has already shared some other comments about cursor position.

Reply:
HTTP POST and GET request will be implemented to support this. Cursor position will need more investigating.